docs/integrations/prometheus/_index.rst

```diff
@@ -60,8 +64,9 @@ other tools can differ, depending on the format and organization of the returned
  Configure Determined
 **********************
 
-Install and run Determined on a cluster. When launching the master instance, enable the Prometheus
-endpoints by adding a flag to the ``master.yaml`` configuration file:
+Install and run Determined on a cluster. When launching the master instance, the Prometheus
+endpoints are enabled by default in versions 0.32.0+. For older versions of Determined, the
+endpoints can be enabled by adding a flag to the ``master.yaml`` configuration file:
```

Comment:
Install and run Determined on a cluster. When launching the master instance, the Prometheus
endpoints are enabled by default in Determined 0.32.0 and later. To manually enable the Prometheus endpoints, add a flag to the ``master.yaml`` configuration file: